src/main/java/org/aigents/nlp/gen/Generator.java

```diff
@@ -148,277 +161,449 @@ public static void main(String[] args) throws IOException {
 		}
 	}
 
-	private static boolean connects(String left, String right) {
-		if (!checkLR(left, right))
-			return false;
-		ArrayList<Rule> leftList = dict.getRule(left), rightList = dict.getRule(right);
-		if (leftList.size() == 0) {
-			System.err.println("Word '" + left + "' not found in dictionary.");
-			System.exit(0);
+	public static HashSet<String> generateSentence(String[] elements) {
+		long start = System.currentTimeMillis();
+		boolean not = false;
+		boolean now = false;
+		HashSet<String> ret = new HashSet<>();
+		int n = elements.length;
+		int[] indexes = new int[n];
+		for (int i = 0; i < n; i++) {
+			indexes[i] = 0;
 		}
-		if (rightList.size() == 0) {
-			System.err.println("Word '" + right + "' not found in dictionary.");
-			System.exit(0);
+		ArrayList<String> w = new ArrayList<>(Arrays.asList(elements));
+		if (w.contains("not") && w.contains("a")) {
```

Comment:
Still issue?

Reply:
Regarding this and the above comments, "not" and "now" are two specific cases that are included; as we discussed earlier, there are a few very specific words that are integrated as edge cases into the algorithm, and "not" and "now" are two of them. Those should be the only instances of hardcodings, though.